utils/service-ip/variables.tf

```diff
@@ -0,0 +1,10 @@
+variable "domain" {
+  description = "Internal domain name of the Kubernetes cluster"
+  type        = string
+  default     = "cluster.local"
+}
+
+variable "service" {
```

Comment:
Is it required variable or could we put default = null ?

Reply:
Yes, it is required. It is the service object we want to get the IP.